### ISSUE TITLE
Message in logs: Installer "Extract *.zip/*.tar.gz" cannot be used to install

### ISSUE THREAD

**Issue by sxa** [adoptium/infrastructure#2477]
[I raised this in a slack thread](https://adoptium.slack.com/archives/C53GHCXL4/p1640260787103500) but since it's not obvious that we have a solution yet I'm copying it here. There is a "JDK installation" role defined in the jenkins Global Tool Config which is intended to install JDK11 on the Azure nodes used for the Linux installer jobs, but it's causing warnings to be thrown in lots of other jobs:
```
Installer "Extract *.zip/*.tar.gz" cannot be used to install "jdk-11.0.13+8" on the node "<blah>"
```
While this was initially seen on the [VPC](https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/) jobs, the role definition in jenkins seems to be in place for various linux systems it's also affecting windows jobs e.g. [create_installer_windows](https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/create_installer_windows/) which is showing the same message (usually repeated six times in succession, repeated multiple times during the logs)

**Comment by sxa** [comment]
Also noted by @karianna in the [sign_build](https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/sign_build) job today.

**Comment by sxa** [comment]
@gdams Can we get some attention on this to clear it up please? It's not good to have these messages across the logs and we should understand why this is happening and endeavour to clear it up and you know how it was introduced. I believe that it originated from work you did for the installer at the end of 2021 and that you'd done some work to clear it up in some places but it still remains an issue in various jobs and you are best place to understand and resolve it.

In addition to the messages seen elsewhere I was working on a prototype jenkins job this week and the ppc64le system being used is failing with this error which seems likely be related given the version number that is specified in the message:
- Building remotely on [dockerhost-skytap-ubuntu2004-ppc64le-1](https://ci.adoptopenjdk.net/computer/dockerhost-skytap-ubuntu2004-ppc64le-1) (build dockerHost dockerBuild linux docker ppc64le)FATAL: Failed to install https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.13_8.tar.gz to /home/jenkins/tools/hudson.model.JDK/jdk-11.0.13_8

**Comment by sxa** [comment]
@gdams Can you clarify the implications if this is removed from the [jenkins tools config](https://ci.adoptium.net/manage/configureTools/) please? Would we need to have JDK11 installed elsewhere on some of the machines? (And does it need to be 11?) Since this was for only one specific purpose it seems suboptimal to have it as something that is triggered for many jobs.

@steelhead31 Will the installer rewrite still require this install?

**Comment by steelhead31** [comment]
I dont believe it is used in the linux package install, unless there's some dependency Im not aware of.

**Comment by sxa** [comment]
We'll get this stripped out after the July release cycle is complete.

**Comment by sxa** [comment]
FYI @steelhead31 I have unchecked the `Install automatically` checkbox so we'll keep an eye open for any fallout. This was the config previously. I'll close this issue and we can re-open if we see any issues:

<img width="903" height="1060" alt="Image" src="https://github.com/user-attachments/assets/dc8b9ecc-1585-47fc-b7be-5487f5514d01" />
